### PR TITLE
[142] Develop a Contact Form Page Using HTML 

### PR DESCRIPTION
**Pull request by DevOge**
unnamed

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
This task supposes to be just purely HTML for the contact form. But now I am seeing CSS and also seeing product page. Kindly follow the instructions given in the task,

**Comment by DevOge** [comment]
> This task supposes to be just purely HTML for the contact form. But now I am seeing CSS and also seeing product page. Kindly follow the instructions given in the task,

Good morning ma. Please where does the CSS apply? The colour and middle stuff? 
The product page link is also part of a given task, i did them both before pushing

**Comment by adaugochi** [comment]
@DevOge You can't do two tasks in one PR. Like I said before, it is one task to one PR

**Comment by adaugochi** [comment]
Kindly work on `[142]` get it merged before proceeding to `[148]`

**Comment by DevOge** [comment]
> I want to believe this is what you are asked to do for this task
> image Kindly just do exactly as it says

> I want to believe this is what you are asked to do for this task
> image Kindly just do exactly as it says

Yes ma, and all the fields are included including the submit button. I googled and saw that CSS had to do with styling, I did not know that before, I was only trying to make the form look a little like what I normally see. Also the instruction doesn't say using only html.  I don't believe extra effort should be discouraged especially when its not violating instructions.
Regardless, I have made the changes you have requested. please review my PR.

**Comment by adaugochi** [comment]
Alright

File: submissions/OgeEjim/html-forms/productpageforcinnamoncakes.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/OgeEjim/html-forms/productpageforcinnamoncakes.html:1]: You should have done product page task before contact form task. Is there a reason why you are redoing it again? Also, the product page shouldn't even be inside `html-forms` folder

**Review by adaugochi — CHANGES_REQUESTED** [review]
Also, all images should be placed inside the `images` folder that is in your `HTML` folder

File: submissions/OgeEjim/html-forms/productpageforcinnamoncakes.html (hunk 1 above)
Comment (DevOge) [reply submissions/OgeEjim/html-forms/productpageforcinnamoncakes.html:1]: oh no, I am not redoing it, I added it to the folder so I could link both pages the way task 148 required. should I take it out?

**Review by adaugochi — APPROVED** [review]

File: submissions/OgeEjim/html-forms/productpageforcinnamoncakes.html (hunk 1 above)
Comment (DevOge) [reply submissions/OgeEjim/html-forms/productpageforcinnamoncakes.html]: > You should have done product page task before contact form task. Is there a reason why you are redoing it again? Also, the product page shouldn't even be inside `html-forms` folder it ought to be in your `HTML` folder

Ma, I have removed the productpage and put the images in a folder

**Pull request merged**